From 7d15c5c44bc3f7db8f40f305be01e60374522df9 Mon Sep 17 00:00:00 2001
From: Gilles DOFFE <g.doffe@gmail.com>
Date: Mon, 14 Aug 2023 03:46:21 +0200
Subject: [PATCH 5/7] cpu/native/periph/qdec: remove useless loop

Signed-off-by: Gilles DOFFE <g.doffe@gmail.com>
---
 cpu/native/periph/qdec.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/cpu/native/periph/qdec.c b/cpu/native/periph/qdec.c
index c735d60cc5..c3a3b8aa91 100644
--- a/cpu/native/periph/qdec.c
+++ b/cpu/native/periph/qdec.c
@@ -71,9 +71,7 @@ int32_t qdec_init(qdec_t qdec, qdec_mode_t mode, qdec_cb_t cb, void *arg)
     }
 
     /* Initialize qdec channels */
-    for (uint8_t i = 0; i < QDEC_NUMOF; i++) {
-        qdecs[qdec] = 0;
-    }
+    qdecs[qdec] = 0;
 
     /* Reset counter and start qdec */
     qdec_start(qdec);
-- 
2.37.0

